litestar/logging/config.py

```diff
@@ -188,23 +200,22 @@ class LoggingConfig(BaseLoggingConfig):
     Filter instance.
     """
     propagate: bool = field(default=True)
-    """If messages must propagate to handlers higher up the logger hierarchy from this logger."""
-    formatters: dict[str, dict[str, Any]] = field(
-        default_factory=lambda: {
-            "standard": {"format": "%(levelname)s - %(asctime)s - %(name)s - %(module)s - %(message)s"}
-        }
-    )
+    """If messages must propagate to handlers higher up the logger hierarchy from this logger.
+
+    .. deprecated:: 2.10.0
+        This parameter is deprecated. It will be removed in a future release. Use `propagate` at the logger level.
+    """
+    formatters: dict[str, dict[str, Any]] = field(default_factory=_get_default_formatters)
+    """A dict in which each key is a formatter and each value is a dict describing how to configure the corresponding
+    Formatter instance. A `standard` formatter is provided.
```

Comment:
```suggestion
    Formatter instance. A ``standard`` formatter is provided.
```

Reply:
for the noun `Formatter` could we link to https://docs.python.org/3/library/logging.html#formatter-objects?

Reply:
Sure, I also added links to Logger, Handler and Filter